authors/apps/articles/views.py

```diff
@@ -0,0 +1,108 @@
+from django.shortcuts import render
```

Comment:
Hello @matthenge ,
Since you are not using this module anywhere, you can remove it from the imports

Reply:
Already fixed